src/examples/10-10CG/example.js

```diff
@@ -9,7 +9,7 @@ const example = {
     alternativePhoneNumber: '8887775544',
     email: 'veteranEmail@email.com',
     vaEnrolled: true,
-    plannedClinic: 'My Clinic',
+    plannedClinic: '635GF',
```

Comment:
Cool so I guess we are going to use facility code? Love that if thats the case so if the name changes we need not to worry.